src/parser.mly

```diff
@@ -86,7 +86,8 @@ expr:
   | expr OR expr              { Binop ($1, Or, $3) }
   | NOT expr                  { Unop (Not, $2) }
   | LPAREN expr RPAREN        { $2 }
-  | ID LPAREN args_opt RPAREN { Call ($1, $3)  }
+  | ID LPAREN args_opt RPAREN { Call ($1, $3) }
+  | LSQUARE elems_opt RSQUARE { ListLit ($2) } 
```

Comment:
Nit: should we move this rule to right below `CharLit`?

Reply:
oh weird I thought I did that, i think i forgot to push lol